mlir-tensorrt/tensorrt/lib/TensorRT/Transforms/RaiseNormalizations.pdll

```diff
@@ -146,6 +146,10 @@ Constraint ReduceSumImpl(val: Value)[{
               (reduceOp.getInput().getType().getRank() - 1)));
 }];
 
+Constraint AvgImpl(op: Op) [{
```

Comment:
I'm sure figuring this out was a joy haha. We need to figure out how to get non-upstream dialect attributes working.